Source/GUI/MainWindow.cpp

```diff
@@ -334,6 +351,19 @@ void MainWindow::onUnhook()
   updateDolphinHookingStatus();
 }
 
+void MainWindow::onAutoLoadLastFileTriggered(const bool checked)
+{
+  if (!checked)
+  {
+    if (!m_watcher->warnIfUnsavedChanges())
+    {
+      return;
+    }
+    m_watcher->clearWatchList();
+    m_watcher->restoreWatchModel(SConfig::getInstance().getWatchModel());
+  }
```

Comment:
Nit: For consistency, the first check could be an early out too.

Reply:
I don't understand what you mean by this

Reply:
Oh... Do you mean flipping this, so if checked == true, return? That way the inner code is not nested?

Reply:
I went with this, pretty sure this is what you meant.
```c++
{
  if (checked)
    return;
  if (!m_watcher->warnIfUnsavedChanges())
    return;
  m_watcher->clearWatchList();
  m_watcher->restoreWatchModel(SConfig::getInstance().getWatchModel());
}
````

Reply:
Yes, exactly. Quite a nitpick, though.